### ISSUE TITLE
Wheel for Raspbery Pi 3 and Python 3.5.2

### ISSUE THREAD

**Issue by halbi** [StudentCV/PyPylon#6]
Hi :)

I would like install pypylon for python 3.5.2 on Raspbian. I tried install wheel cp34-cp34m but I obtain errors.

Thanks for help :)

**Comment by oliver02** [comment]
Hi, 
try the cp35-cp35m .whl. That should work out.

**Comment by halbi** [comment]
This one: pypylon-1.0.7+pylon5.0.5.9000-cp35-cp35m-linux_x86_64.whl ?

**Comment by oliver02** [comment]
Yes. after 'cp' the number specifies your python version.

**Comment by halbi** [comment]
Yes, I know it. What about linux_x86_64.whl? It is a wheel for traditional processors. Raspbery PI uses ARM. I found information that package for RPI should be named linux_armv7l.
